edgexfoundry/edgex-compose
# build: Unpin patch on 3rd party versions and use latest

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
run `make pull` from top folder
run `make run` from top folder
Verify all service bootstrapped
Change app-rules-engine logging to DEBUG in Consul
Verify logs show events flowing
Test version/ping end point via API gate way.
Verify UI is seeing the following
- 2 device services
- 9 devices
- 9 device profiles
